templates/console_readme_suffix.md

```diff
@@ -0,0 +1,43 @@
+## Getting started
+
+For complete information [see this article](https://docs.apify.com/platform/actors/development#build-actor-at-apify-console). In short, you will:
+
+1. Build the Actor
+2. Run the Actor
+
+## Pull the Actor for local development
+
+If you would like to develop locally, you can pull the existing Actor from Apify console using Apify CLI:
+
+1. Install `apify-cli`
+
+    **Using Homebrew**
+
+    ```
+    brew install apify/tap/apify-cli
+    ```
+
+    **Using NPM**
+
+    ```
+    npm -g install apify-cli
+    ```
+
+2. Pull the Actor by its unique `<ACTID>`, which is:
+    - name of the Actor to pull (e.g. "apify/hello-world")
+    - or ID of the Actor to pull (e.g. "E2jjCZBezvAZnX8Rb")
+
+    This command will copy the Actor into the current directory on your local machine.
+
+    ```
+    apify pull <ACTID>
```

Comment:
It's not ACTID, but Actor ID. The fact that that it's named `actId` on the API is simply because 5 years ago Actors were called Acts, but we haven't been using this name since. ACTID is not a short for (unique name OR id).

Please guide the users where to get the ID / **unique** name (yes, not a name, we have title, unique name and ID) or add a screenshot please. This is still not good enough for the users.

![CleanShot 2023-06-14 at 11 28 50@2x](https://github.com/apify/actor-templates/assets/23726914/9c556dee-d512-4105-8863-d9082a29b110)

![CleanShot 2023-06-14 at 11 29 56@2x](https://github.com/apify/actor-templates/assets/23726914/e7302e80-ab74-453b-be42-0494ddd85f2e)

Reply:
Ok ok thx I will try to add the instructions. Unforunately the screenshots are be displayed in the Web IDE readme